README.md

```diff
@@ -133,10 +134,39 @@ CycleTimeMs - Cycle time of the application.  All tags will be posted at this sp
 
 PostDuplicateTagValues - Controls when datapoints are logged.  If set to true, datapoints will always be cyclicly logged. If set to false, datapoints will only be logged on change of value.
 
+OMF - set to true if you have OMF enabled in your PIWEBAPI configuration tool, otherwise set this to false for legacy systems. See more info on OMF below.
+
 ### TagList
 
 TagList - List of eWON tags that should be connected to the OSIsoft PI server.  If PI Points with (non case sensitive) matching names do not exist on the PI server they will be created automatically.
 
 ## Customizing the application
 
-If you wish to modify, debug, or rebuild the application the toolkit and documentation is available here https://developer.ewon.biz/content/java-0. The instructions for setting up your development environment are here  https://developer.ewon.biz/system/files_force/AUG-072-0-EN-%28JAVA%20J2SE%20Toolkit%20for%20eWON%20Flexy%29.pdf?download=1
\ No newline at end of file
+If you wish to modify, debug, or rebuild the application the toolkit and documentation is available here https://developer.ewon.biz/content/java-0. The instructions for setting up your development environment are here  https://developer.ewon.biz/system/files_force/AUG-072-0-EN-%28JAVA%20J2SE%20Toolkit%20for%20eWON%20Flexy%29.pdf?download=1
+
+## OMF Support
+
+The OSIsoft connector now has the ability to support OMF. To enable OMF follow the following steps:
+1. Download the latest version of PIWEBAPI - PIWEBAPI 2019 or later is required.
+   1. Install the new software.
+1. Enable CSRFDefence.
+   1. Open Pi System Explorer.
+   1. Expand elements -> osisoft -> PI Web API -> 'computer name' -> System Configuration
+   1. Click on attributes and find the field for EnableCSRFDefence.
+   1. Change the value to true and click the 'check in' button at the top.
+1. Run the PI Web API Admin Utility.
+   1. In the OMF services section of the configuration tool, click edit.
+   1. Make sure there is a green checkmark next to every field.
+   1. IMPORTANT: The PI Data Archive Server can disconnect for various reasons. Make sure this is connected if you have a connection issue.
+   1. Hit confirm and finish running the utility.
```

Comment:
change hit to select or click. you use click above so probably just be consistent.